examples/server/server.cpp

```diff
@@ -3907,6 +3907,21 @@ int main(int argc, char ** argv) {
         res_ok(res, {{ "success", true }});
     };
 
+    const auto handle_api = [&ctx_server, &res_ok](const httplib::Request &, httplib::Response & res) {
```

Comment:
maybe naming like this to make it clearer ?

```suggestion
    const auto handle_api_show = [&ctx_server, &res_ok](const httplib::Request &, httplib::Response & res) {
```